documentation/asciidoc/computers/camera/camera_usage.adoc

```diff
@@ -1,13 +1,9 @@
-== Introducing the Raspberry Pi cameras
+This documentation describes how to program and use supported camera modules with our software tools. All Raspberry Pi cameras can record high-resolution photographs and full HD 1080p video (or better) with our software tools.
 
-There are now several official Raspberry Pi camera modules. The original 5-megapixel model was https://www.raspberrypi.com/news/camera-board-available-for-sale/[released] in 2013. It was followed by an 8-megapixel https://www.raspberrypi.com/products/camera-module-v2/[Camera Module 2], which was https://www.raspberrypi.com/news/new-8-megapixel-camera-board-sale-25/[released] in 2016. The latest camera model is the 12-megapixel https://raspberrypi.com/products/camera-module-3/[Camera Module 3] which was https://www.raspberrypi.com/news/new-autofocus-camera-modules/[released] in 2023. The original 5MP device is no longer available from Raspberry Pi. 
+Raspberry Pi produces several official camera modules, including the original 5-megapixel Camera Module 1, the 8-megapixel https://www.raspberrypi.com/products/camera-module-v2/[Camera Module 2], and the 12-megapixel https://raspberrypi.com/products/camera-module-3/[Camera Module 3]. The Camera Module 3 also comes in a wide field-of-view model. We produce infrared versions of each, except for the Camera Module 1, which has been discontinued.
 
-Additionally a 12-megapixel https://www.raspberrypi.com/products/raspberry-pi-high-quality-camera/[High Quality Camera] with CS- or M12-mount variants for use with external lenses was https://www.raspberrypi.com/news/new-product-raspberry-pi-high-quality-camera-on-sale-now-at-50/[released in 2020] and https://www.raspberrypi.com/news/new-autofocus-camera-modules/[2023] respectively. There is no infrared version of the HQ Camera.
+Additionally, we make a 12-megapixel https://www.raspberrypi.com/products/raspberry-pi-high-quality-camera/[High Quality Camera] with CS- or M12-mount variants for use with external lenses. There is no infrared version of the HQ Camera.
```

Comment:
Should this also mention the Global Shutter camera? https://www.raspberrypi.com/products/raspberry-pi-global-shutter-camera/

Reply:
Absolutely. I think I'm just going to make this a list.